pyqt-apps/siriushla/as_di_scrns/base.py

```diff
@@ -443,7 +443,7 @@ def maxHeightChannel(self, value):
 
     def mousePressEvent(self, ev):
         pos = ev.pos()
-        pos_scene = self.view.mapSceneToView(pos)
+        pos_scene = self.view.vb.mapSceneToView(pos)
```

Comment:
Is there a bug here? 
I tested with this new code and this does not work for me.

Reply:
This only works with `pydm>=1.7.3`. For previous versions, the old code was correct.